src/helpers/misc.ts

```diff
@@ -1,11 +1,13 @@
 //------------------------------------------------------------------------------
 // Miscellaneous
 //------------------------------------------------------------------------------
-import { NEWLINE } from "../constants";
+import { FORBIDDEN_IN_EXCEL_REGEX, NEWLINE } from "../constants";
```

Comment:
we might want to take this opportunity to rename this constant ? The name kinda sucks, it doesn't tell you what are forbidden (characters) or where (in sheet names).